### ISSUE TITLE
Where's the SoftMax?

### ISSUE THREAD

**Issue by roryhr** [facebook/fb.resnet.torch#46]
I'm curious why the SoftMax isn't part of the model definitions.

**Comment by colesbury** [comment]
the CrossEntropyCriterion includes LogSoftMax
